go.mod

```diff
@@ -1,35 +1,45 @@
-module github.com/oapi-codegen/fiber-middleware
+module github.com/oapi-codegen/fiber-middleware/v2
```

Comment:
I'd say let's bump this to v3 if we're doing this (so it matches the major version of Fiber)